benchs/Fake/Text.hs

```diff
@@ -8,6 +8,7 @@ module Fake.Text
     , filter
     , reverse
     , decimal
+    , double
```

Comment:
I don't really get the purpose of Fake.Text - perhaps it needs to say at the top of the module why it might be useful? Or perhaps because its in benchs that's not a problem.